### ISSUE TITLE
multimc unsupported download type browser DNS techpack 6.6.x

### ISSUE THREAD

**Issue by bensuperpc** [MultiMC/Launcher#4374]
### Operating System

Windows

### Description of bug

When trying to add an instance of DNS techpack **6.6.1** , there is this error `Unsupported download type: browser` :
![fgfgdfgdfgdhdh](https://user-images.githubusercontent.com/28039927/146697587-0a098a23-c3c1-4cc9-b65e-16f79d428c2e.PNG)

### Steps to reproduce

Add an instance of DNS techpack version **6.6.1**, It works with other versions.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by phit** [comment]
yeah, browser downloads were never added to multimc's atlauncher implementation, see the notes in https://github.com/MultiMC/Launcher/pull/3314
